nano/core_test/confirmation_height.cpp

```diff
@@ -763,23 +735,14 @@ TEST (confirmation_height, pending_observer_callbacks)
 
 	node->confirmation_height_processor.add (send1->hash ());
 
-	while (true)
-	{
-		if (node->pending_confirmation_height.size () == 0)
-		{
-			break;
-		}
-	}
-	node->confirmation_height_processor.add (send.hash ());
-	system.deadline_set (5s);
-	while (node->ledger.cache.cemented_count < 3)
+	system.deadline_set (10s);
+	while (1 != node->stats.count (nano::stat::type::http_callback, nano::stat::detail::http_callback, nano::stat::dir::out) || 1 != node->ledger.stats.count (nano::stat::type::observer, nano::stat::detail::all, nano::stat::dir::out))
```

Comment:
Probably for consistency with other changes `node->stats.count () != 1` ?

Reply:
Yep, thanks!